### PR TITLE
PAI Fixes

### LINKED ISSUE #5529: (WEB REPORT BY: acedick REMOTE: 198.245.63.50:7777) pAI new update issues
>Revision c81a7bc6e4183ac6172c98b7729cdeb8e330b228

>General description of the issue
The new medical package doesn't actually allow you to access records, clicking on names only brings you back to the main pAI menu.

The wire jack returns a "WARNING: Unable to interface with X." when jacking into airlocks and using options, even when the action succeeds. Also not sure if bug or intended feature, when it does this the airlock closes again almost instantly.

Also minor feature request tacked onto this, the ability for the medical package to scan the person holding the pAI again was removed with the medical suite, it's actually critically useful.

>What you expected to happen
Clicking on the names of people on the medical records list to see their records.

When accessing airlocks to have them open for longer than a second.
>What actually happened
Records returned you back to the menu without showing you anything.
Door closes almost instantly, still enough to let someone inside, also returns an error every time.

>Steps to reproduce if possible
-download medical package and attempt to click on someone's name for their records

-have someone use you on an airlock to have you hack it
-on the door open menu, attempt to open the door

### LINKED ISSUE #5582: (WEB REPORT BY: acedick REMOTE: 198.245.63.50:7777) New pAI flashlight doesn't work
>Revision c8dad8500b5d923af3e311a4ba8ba501294f8639 

>General description of the issue
pAI flashlight doesn't produce light.

>What you expected to happen
Light to be produced as the description describes

>What actually happened
With no lights on, the area stayed pitch black even turning it on and off.

>Steps to reproduce if possible
>Buy pAI flashlight app
>activate said app
>light doesn't increase regardless of where it's put, even if in hand

### LINKED ISSUE #5607: (WEB REPORT BY: headless1 REMOTE: 198.245.63.50:7777) PAI sec package
>Revision (Should be above if you're viewing this from ingame!)
84a4ea3edbf290fc561e677b9b04a19a678d51c1

>General description of the issue
The Security Package for PAI's doesn't appear to work.

>What you expected to happen
When I click on the name of a person in the sec package menu, I expect to see their sec records page.

>What actually happened
The page is blank, devoid of useful information.

>Steps to reproduce if possible
Be a PAI with a security package and try clicking on a person's name in the package menu.

### PR DESCRIPTION
**Pull request by Kurfursten**
- Fixed accessing medical and security records as a pAI, fixes #5607, fixes , fixes #5607
- Fixed wirejacking doors as a pAI by removing the AI menu and instead just making it force the door open -- short term solution until I can figure out why it keeps autoclosing, fixes #5529
- Brightness Enhancer bugs resolved, fixes #5582
- Allows you to access host bioscan once again

I'm mostly posting this so I can take a break from working on pAI fixes, I've been at some of these (why pAIs can't ClickOn, why pAIs can't use the AI door menu properly) for several days now, relentlessly. Maybe someone else can help me out on the matter. If not, I'll pick it up in a bit.

### CODE REVIEW

**Comment by N3X15** [comment]
Conflicts

**Comment by Kurfursten** [comment]
Fixed conflicts and as a bonus, you can access host bioscan once again

**Comment by Kurfursten** [comment]
Why is this still tagged as conflicts? Why wasn't it merged? It's not a feature, it's a fix.

**Comment by Kurfursten** [comment]
MERGE THE FIX

OR SUFFER MY CURSE

**Comment by Probe1** [comment]
Reviewed, tested, working as intended.

@N3X15 
@clusterfack 
@d3athrow 
@Anyone merge this already so I can update the wiki.

File: code/modules/mob/living/silicon/pai/software.dm [hunk 1]
```diff
@@ -481,10 +481,11 @@
 	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""])  \\/mob/living/silicon/pai/proc/softwareMedicalRecord() called tick#: [world.time]")
 	var/dat = ""
 	if(src.subscreen == 0)
+		dat += "<a href='byond://?src=\ref[src];software=medicalsupplement;sub=2'>Host Bioscan</a><br>"
```
Comment (Probe1) [inline code/modules/mob/living/silicon/pai/software.dm]: [14:47:51] [14:47:51] <Ririchiyo> in software.dm at line 484
[14:48:54] [14:48:54] <Ririchiyo> do we have links with byond://?

Comment (N3X15) [inline code/modules/mob/living/silicon/pai/software.dm]: Remove the `byond://` and it'll work.

**Pull request merged**